app/controllers/concerns/commentable.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Commentable
+  extend ActiveSupport::Concern
+
+  def create
+    @comment = @commentable.comments.new(comment_params)
+
+    if @comment.save
+      redirect_to @commentable, notice: t('controllers.common.notice_create', name: Comment.model_name.human)
+    else
+      render @commentable
+    end
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:content).merge(user_id: current_user.id)
```

Comment:
オブジェクトを使うこともできるので、そうしてみてください 💪 

```suggestion
    params.require(:comment).permit(:content).merge(user: current_user)
```